netci.groovy

```diff
@@ -1962,6 +1980,7 @@ Constants.allScenarios.each { scenario ->
                                 }
                                 break
                             case 'default':
+                            case 'pri1':
```

Comment:
e.g.,
```
case 'pri1':
                    def isWindowsArmJob = (os == "Windows_NT" && architecture in validWindowsNTCrossArches)
                    if (isWindowsArmJob == false) {
                        return
                    }
                    break
```

Seems like we should clean this up, too, by making validWindowsNTCrossArches a global constant (since it is defined several places), and adding a function:
```
...
    // This is the set of architectures
    def static architectureList = ['arm', 'armlb', 'x86_arm_altjit', 'x64_arm64_altjit', 'arm64', 'x64', 'x86']

    // The set of architectures that do cross-compilation builds on Windows.
    def static WindowsCrossArchitectures = ['arm', 'armlb', 'arm64']
}

...

def static isWindowsArmJob(def os, def architecture)
{
    return (os == "Windows_NT" && architecture in Constants.WindowsCrossArchitectureList)
}
```